packages/notebook/src/panel.ts

```diff
@@ -157,16 +146,6 @@ export class NotebookPanel extends DocumentWidget<Notebook, INotebookModel> {
     super.dispose();
   }
 
-  /**
-   * Handle `'activate-request'` messages.
-   */
-  protected onActivateRequest(msg: Message): void {
-    super.onActivateRequest(msg);
-
-    // TODO: do we still need to emit this signal? Who is using it?
```

Comment:
I assume this means that there really was nothing consuming the `activated` signal, right?

Reply:
Correct, thanks for the review!